portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,21 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> commentsStorage = new ArrayList<>();
```

Comment:
run clang format on this for spacing too

Reply:
Yes, I have just adjusted the format again according to clang format. Thank you for pointing it out!